webui/src/components/editrecord.jsx

```diff
@@ -168,7 +168,14 @@ const EditRecord = React.createClass({
             }
         }
         console.assert(!Array.isArray(value));
-        r = value !== undefined ? r.setIn(path, value) : r.deleteIn(path);
+        let trimmedValue;
+        if(typeof value === 'string' || value instanceof String) {
+            trimmedValue = value.replace(/^\s+/, '').replace(/\s+$/, ' ') ;
```

Comment:
Using trim() here doesn't allow to have multiple words because the state updates after pressing any key and if it is a space, it will be removed by trim(). Then user need to write them like a compound word and then move the cursor back and add a space which I think it's annoying